appengine-skeleton-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,10 +1,7 @@
-#set( $symbol_pound = '#' )
-#set( $symbol_dollar = '$' )
-#set( $symbol_escape = '\' )
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
     <application>${application-id}</application>
-    <version>${symbol_dollar}{appengine.app.version}</version>
+    <version>${appengine.app.version}</version>
```

Comment:
Not a big deal, but
This file will be filtered. If you add appengine.app.version to the archetype metadata, this value will be replaced with the specified value, which I don't think the expected behavior.